spec/features/cards_spec.rb

```diff
@@ -0,0 +1,106 @@
+require 'rails_helper'
+
+RSpec.describe "Cards", type: :feature do
+  context 'when create new card' do
+    it 'when successful' do
+      visit new_card_path
+      within('form') do
+        fill_in 'Оригинальный текст', with: 'ace'
+        fill_in 'Перевод текста', with: 'race'
+      end
+      click_button 'Create'
+      expect(page).to have_content('Перевод')
+    end
+
+    it 'when fails' do
+      visit new_card_path
+      within('form') do
+        fill_in 'Оригинальный текст', with: 'ace'
+        fill_in 'Перевод текста', with: 'ace'
+      end
+      click_button 'Create'
+      expect(page).to have_content('Original text Original text can\'t be translated')
+    end
+  end
+
+  context 'when update card' do
+    let!(:card) { create(:card) }
+
+    before do
+      visit edit_card_path(card)
+    end
+
+    it 'when successful' do
+      within('form') do
+        fill_in 'Оригинальный текст', with: 'gnom'
+        fill_in 'Перевод текста', with: 'guru'
+      end
+      click_button 'Create'
+      expect(page).to have_content('guru')
+    end
+
+    it 'when fails' do
+      within('form') do
+        fill_in 'Оригинальный текст', with: ''
+      end
+      click_button 'Create'
+      expect(page).to have_content('be blank')
+    end
+  end
+
+  context 'when destroy card' do
+    let(:card) { create(:card) }
+
+    it 'when successful' do
+      card.original_text = 'word'
+      card.translated_text = 'other'
+      visit cards_path
+      expect { click_link 'Delete' }.to change(Card, :count).by(-1)
+    end
+  end
+
+  context 'when model' do
+    it 'when word correct' do
+      visit new_card_path
+      within('form') do
+        fill_in 'Оригинальный текст', with: 'дом'
+        fill_in 'Перевод текста', with: 'ДоМ'
+      end
+      click_button 'Create'
+      expect(page).to have_content('Original text can\'t be translated')
+    end
+  end
+
+  describe 'when training' do
+    let(:card) { create(:card) }
+
+    before do
+      card.update(review_date: '2018-12-01')
```

Comment:
Please do not use real dates, work with relative dates, like a `(Date.current + 1.day).to_s #=>  "2018-12-22"`